tests/job_tests/corpus/files/test_job.corpus.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<corpus name="job-testing">
+  <speaker-description name="116">
+    <gender>male</gender>
+  </speaker-description>
+  <recording name="116-288045-0000" audio="sample.wav">
+    <speaker name="116"/>
+    <segment name="116-288045-0000" start="0.0000" end="10.6500">
+      <orth> AS I APPROACHED [non-speech] THE CITY, I HEARD BELLS RINGING [noise] [noise] AND A    LITTLE LATER I FOUND THE STREETS ASTIR WITH THRONGS OF WELL DRESSED PEOPLE IN FAMILY GROUPS WENDING THEIR WAY HITHER AND THITHER!!! </orth>
```

Comment:
How important are the whitespace after the A? 
It does not matter for approval, but for the `CorpusReplaceOrthFromReferenceCorpus` test  these must not be there and if you "dont need them" we could maybe merge the test corpora? Not sure if this makes sense, or if each test should have its own files.

Reply:
I think these whitespaces are there to explicitly test the fix whitespace option, so I think we should keep it. 
It is in general, I think, fine to have separate files per test case.

Reply:
the extra white space is there for a reason. to test the flag `fix_whitespace`